ramls/item.json

```diff
@@ -128,6 +132,34 @@
             "type": "string",
             "description": "Text to display"
           },
+          "source": {
+            "type": "object",
+            "description": "The user who added/updated the note. The property is generated by the server and needed to support sorting.",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The id of the user who added/updated the note"
```

Comment:
Does this point to /users/{id}, if yes, then it should be mentioned in the description.

Reply:
Yes it does. Updated the description.

Reply:
Thanks for adding it to the description of "source" in line 138, can you also add it to the "id" description in line 142?

Reply:
done